conf/application.conf

```diff
@@ -97,3 +97,5 @@ contact-frontend {
 
 # to learn why this was included: https://confluence.tools.tax.service.gov.uk/display/TEC/2016/03/14/Setting+Security+Headers+in+frontend+services
 play.filters.headers.contentSecurityPolicy= "default-src 'self' 'unsafe-inline' localhost:9032 localhost:9250 www.google-analytics.com data:"
+
+google-analytics.token: FAKE_GA_TOKEN_CHANGE_ME_OR_DELETE
```

Comment:
Needed to check the loading of the GA script locally. I may not need this on the master.